e2e/README.md

```diff
@@ -1,19 +1,27 @@
 We have e2e tests using playwright
 
+Go to the e2e directory
+
+`molgenis-emx2-repository\e2e`
```

Comment:
this is a e2e readme for all the apps , maybe move the directory stuff te a separate folder , with readme

Reply:
The idea is to point to the e2e directory for all the apps, running 'yarn e2e' in the project root is not going to do anything. Yes the readme may already be in the correct directory but the user may not be.

Reply:
a now i see , you don't mean the 'directory' app but the folder named 'e2e'. suggest using the term folder to avoid confusion, you als add a link to avoid confusion, 'molgenis-emx2-repository' may be somewhat confusing ( i guess you mean the project root folder) .